app/src/components/integrations/elements/integrationSettings/connectionSection/connectionSection.jsx

```diff
@@ -197,7 +197,11 @@ export class ConnectionSection extends Component {
         >
           <div className={cx('integration-info-block')}>
             <div className={cx('sub-header-block')}>
-              <div className={cx('general-info')}>
+              <div
+                className={cx('general-info', {
+                  blocked: blocked && availableProjectIntegrations.length > 0,
```

Comment:
I think that in these case we can avoid of using availableProjectIntegrations.length > 0. 
But it is not mandatory to fix.

Reply:
without this, styling will be improper

Reply:
Okay, may be you are right.